app/src/ui/nav-bar/nav-bar.js

```diff
@@ -55,7 +56,7 @@ export default function NavBar({
           icon={titleIcon}
           data-test-nav-title
         >
-          {title}
+          <Link to={`/${roomCode}`}>{title}</Link>
```

Comment:
This component is also used in modals where this link wouldn't make sense. We could instead add this link around the room code itself, which isn't rendered in the modals.